README.md

```diff
@@ -23,9 +23,9 @@ python -m pip install .
 
 ## CallHome Dataset
 
-Go [https://ca.talkbank.org/access/CallHome](here), select the conversation language, create account, then you can download the "media folder". There you can find the .cha files, which contain the transcriptions.
+Go [https://ca.talkbank.org/access/CallHome](here), select the conversation language, create account, then you can download the "media folder". There you can find the .cha files, which contain the transcriptions. You will need to pre-process this data in combination with the Callhome translations dataset, which includes part of the pre-processing scripts. The README under ./scripts/callhome of this repo contains more information.
```

Comment:
There isn't a README in scripts/callhome currently.

Reply:
Ah yes, sorry - adding it to my next commit